pyproject.toml

```diff
@@ -53,6 +53,7 @@ test = [
 dev = [
   "pytest >=6",
   "pytest-cov >=3",
+  dvc[gdrive]
```

Comment:
I think this just needs to be in quotes. Also good to have a trailing comma.

If you want to test that the pyproject.toml works, you can create a fresh python environment, and then pip install from the directory:

```
pip install /path/to/openlifu-python-repo
```

Adding `[dev]` will install with these options:

```
pip install /path/to/openlifu-python-repo[dev]
```